eppo_client/eval.py

```diff
@@ -0,0 +1,93 @@
+from typing import Dict, Optional, Union
+from eppo_client.sharding import Sharder
+from eppo_client.models import Flag, Range, Shard, Variation
+from eppo_client.rules import matches_rule
+from dataclasses import dataclass
+import datetime
+
+
+@dataclass
+class FlagEvaluation:
+    flag_key: str
+    subject_key: str
+    subject_attributes: Dict[str, Union[str, float, int, bool]]
```

Comment:
optional?

Reply:
We always return a dict here -- it might just be empty. I imagine that's easier to deal with downstream than having either nulls or dictionaries with values. Open to feedback